cpp/velox/compute/VeloxInitializer.cc

```diff
@@ -89,6 +89,9 @@ const std::string kVeloxUdfLibraryPaths = "spark.gluten.sql.columnar.backend.vel
 const std::string kMaxSpillFileSize = "spark.gluten.sql.columnar.backend.velox.maxSpillFileSize";
 const std::string kMaxSpillFileSizeDefault = std::to_string(20L * 1024 * 1024);
 
+// backtrace allocation
+const std::string kBacktraceAllocation = "spark.gluten.backtrace.allocation";
```

Comment:
Do we want to add this configuration into [README](https://github.com/oap-project/gluten/blob/main/docs/Configuration.md?plain=1#L50)?

Reply:
We may add introduction info into dev/docs later, Only developers will be concerned about this debug feature.